obspy/clients/arclink/client.py

```diff
@@ -210,7 +210,7 @@ def _bye(self):
         self._write_ln('BYE')
         self._client.close()
 
-    def _fetch(self, request_type, request_data, route=True):
+    def _fetch(self, request_type, request_data, route=False):
```

Comment:
Maybe this one should stay `route=True` as a safe default? We can still manually use `route=False` in all metadata requests below, like you propose.

Reply:
@megies: Sorry for following up late on this. Just a note for the record.

I had/have not a too strong opinion on this, especially because in practice this really should not have any relevance. All calls now specify the parameter explicitly. 

However, there was some reasoning on changing `route=False`, because `route=True` implies recursion. I judged that for this lower level routine `route=False` was safer wrt. to avoiding infinite recursions. 

Now, propose not to touch it anymore. I may look into this part at some point again to get it cleaner and more linear.